astropy/visualization/mpl_normalize.py

```diff
@@ -1,14 +1,18 @@
 """
-Normalization class for Matplotlib that can be used to produce colorbars.
+Normalization class for Matplotlib that can be used to produce
+colorbars.
 """
 
 from __future__ import division, print_function
-
 import numpy as np
 from numpy import ma
+from .interval import (PercentileInterval, AsymmetricPercentileInterval,
+                       ManualInterval, MinMaxInterval)
+from .stretch import (LinearStretch, SqrtStretch, PowerStretch, LogStretch,
+                      AsinhStretch)
 
 try:
-    import matplotlib  # pylint: disable=W0611
+    import matplotlib    # pylint: disable=W0611
```

Comment:
I think it's *at least* two spaces.  I think this was inadvertent, so I could change it back.